Automattic/vip-design-system
# Button > Disabled > Remove hover effect and lessen the opacity

## Description

Disabled button should follow [Figma](https://www.figma.com/file/ugXpCbWjNPoBxz7y8NBgsK/User-Management?type=design&mode=design&t=Ng6kt6mtzrOgEmh0-0)
Figma:
![image](https://github.com/Automattic/vip-design-system/assets/2224344/82298644-aefe-497a-a255-b4dac55fd7f0)

Actual:
Light
![image](https://github.com/Automattic/vip-design-system/assets/2224344/af637ef4-5034-4c17-a6fa-659a3a0e63b2)

Dark
![image](https://github.com/Automattic/vip-design-system/assets/2224344/c5c4d3ff-6768-4cbd-bfc1-b8db716f2385)

## Checklist


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Pull down PR.
2. `npm run storybook`.
3. Button > Disabled > Hover > verify no hover effect > opacity should be less
